src/components/order-service/order-service.component.ts

```diff
@@ -2,7 +2,8 @@ import {OrderServiceController} from './order-service.controller';
 
 export const orderService: angular.IComponentOptions = {
   bindings: {
-    service: '<'
+    serviceClass: '<',
```

Comment:
Hmm...I thought the ordering panel/service would be able to handle both.  I thought there were only a few differences such as imageStreams have a 'versions' dropdown, serviceClasses have 'plans'  -both have params, right?